lib/active_admin/views/index_as_table.rb

```diff
@@ -194,7 +194,7 @@ def default_actions(*args)
               links << link_to(I18n.t('active_admin.edit'), edit_resource_path(resource), :class => "member_link edit_link")
             end
             if controller.action_methods.include?('destroy') && authorized?(ActiveAdmin::Auth::DESTROY, resource)
-              links << link_to(I18n.t('active_admin.delete'), resource_path(resource), :method => :delete, :data => {:confirm => I18n.t('active_admin.delete_confirmation')}, :class => "member_link delete_link")
+              links << link_to(I18n.t('active_admin.delete'), resource_path(resource), :method => :delete, 'data-confirm' => I18n.t('active_admin.delete_confirmation'), :class => "member_link delete_link")
```

Comment:
Line is too long. [205/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.